doc/whats_new/v0.24.rst

```diff
@@ -53,6 +53,12 @@ Changelog
   unless data is sparse.
   :pr:`17396` by :user:`Jiaxiang <fujiaxiang>`.
 
+- |Enhancement| :func:`datasets.fetch_covtype` now now supports optional
+  argument `as_frame`; when it is set to true, the returned Bunch object's
+  `data` and `target` members are in pandas DataFrame format, and the Bunch
```

Comment:
In this case, `target` is a pandas series.

Reply:
I updated this, will merge when test pass

Reply:
@thomasjpfan Thank you sir!